Microservices liveness probes with help of Prometheus (#1173)
GoogleCloudPlatform/bank-of-anthos: extras/prometheus/oss/README.md

> # OSS Prometheus on GKE cluster

Reviewer: ```suggestion
# OSS Prometheus on a GKE cluster
```

> This directory contains files related to configuring probing of Bank of Anthos' microservices with OSS Prometheus on GKE.

Reviewer: ```suggestion
This directory contains files essential to setup liveness probes for Bank of Anthos' microservices running in GKE using OSS Prometheus.
```

> * `values.yaml` - Configuration file to [OSS Prometheus helm chart](https://github.com/bitnami/charts/tree/main/bitnami/kube-prometheus)

Reviewer: ```suggestion
* `values.yaml` - contains the configurations for the [OSS Prometheus helm chart](https://github.com/bitnami/charts/tree/main/bitnami/kube-prometheus)
```

> * `probes.yaml` - File contains Probe custom resource manifests for each microservice

Reviewer: ```suggestion
* `probes.yaml` - contains the Probe custom resource definitions per microservice. One monitoring configuration per microservice is defined.
```

> * `rules.yaml` - File contains PrometheusRule custom resource

Reviewer: ```suggestion
* `rules.yaml` - contains the PrometheusRule custom resource definition.
```

> * `alertmanagerconfig.yaml` - File contain AlertmanagerConfig custom resource with configuration of Alertmanager to send notifications to Slack channel via Webhook URL.

Reviewer: ```suggestion
* `alertmanager.yaml` - contains configuration of AlertmanagerConfig custom resource . This resource defines the configurations necessary to send notifications to a Slack channel via Webhook URL.
```
